scripts/run-celery-beat.sh

```diff
@@ -15,12 +15,5 @@ echo
 # Postgresql makes the port available after the db is available, but wait some more to be sure. This can be removed if the real pg_isready is used.
 sleep 2
 
-echo "Waiting for http://osidb-service:8000/osidb/healthy to become available"
-while ! ( { curl -f http://osidb-service:8000/osidb/healthy >/dev/null 2>&1 || exit 1 ; } ) ; do echo -n "." ; sleep 2 ; done
-echo
-
-
-# custom run script for starting osidb celery-beat service in osidb-stage and osidb-prod environments.
-
-rm -f /tmp/celery_beat.pid
-exec celery -A config beat
+# Reuse existing standalone version
```

Comment:
Why was the healthy check removed? I don't see it in the standalone version either.

Reply:
I don't think it's necessary, the celery pods should be able to run independently of the web service as they don't depend on it.